src/database/exiles_database.py

```diff
@@ -90,12 +93,36 @@ def get_pending_unexiles() -> list[PendingExile]:
         return [PendingExile(*x) for x in res]
 
 
-def get_user_exiles(user_id) -> List[tuple]:
+def get_user_active_exile(user_id) -> PendingExile:
     conn = DatabaseConnection()
 
     with conn.get_cursor() as cursor:
         query = """
-        SELECT e.exileID, e.reason, e.startTimestamp, e.endTimestamp, e.exileStatus
+        SELECT e.exileID, u.userID, u.discordUserID, e.endTimestamp
+        FROM exiles e
+        JOIN users u ON e.userID = u.userID
+        WHERE u.userID = %s AND  e.exileStatus = %s 
+        LIMIT 1;
+        """
+
+        params = (user_id, ExileStatus.TIMED_EXILED)
+
+        cursor.execute(query, params)
+        res = cursor.fetchone()
+
+        if res is not None:
+            return PendingExile(*res)
+        else:
+            return None
+
+
+#   exile_id, user_id, discord_id, reason, exile_status, start_timestamp, end_timestamp
+def get_user_exiles(user_id) -> Exile:
```

Comment:
Can you update the return type here to `List[Exile]`

Reply:
Fixed in latest